### ISSUE TITLE
LeetCode Debug option is not showing up

### ISSUE THREAD

**Issue by Debanil1996** [wangtao0101/vscode-debug-leetcode#26]
## 🐛 Bug Report

I noticed that the Debug option in leetcode is not showing up. Either it is not refreshing or not showing up .I using Macbook Pro 

## To Reproduce

To reproduce .You uninstall and install .Then suddenly the debug option is not at all showing up

## Extension Output

Paste here the LeetCode extension log from output channel.

Guidance: Press `Ctrl+Shift+U`, and toggle the channel to `Debug LeetCode`.

## Your Environment

-   _os_:
-   _extension settings_:
-   _nodejs version_:
-   _vscode version_:
-   _extension version_:

**Comment by Ankan1998** [comment]
yes, I have the same issue. Sometimes it shows up and sometimes it doesn't. I am also using a MacBook pro. Attaching a screenshot for better clarity.

![Screen Shot 2022-10-13 at 4 01 48 PM](https://user-images.githubusercontent.com/36896102/195574836-faf487d1-6241-47eb-9d53-33fc45471423.png)

**Comment by IiMM0811** [comment]
win10  I have the same issue;  it never shows up 
![image](https://user-images.githubusercontent.com/61083617/222969801-42480b90-6887-4e96-8d1d-c45afbdbca2d.png)

**Comment by yippee0539** [comment]
win10 I also have the same issue: some shows up but others don't

**Comment by MODOUser** [comment]
same, m2 pro, it never shows up
use python
